app/views/csvfiles/new.html.erb

```diff
@@ -10,27 +10,27 @@
 <%= stylesheet_link_tag '/lib/simple-data-grapher/examples/upload_file.css'%>
 <div id='first'></div>
 <script>
-    var headerContainer = document.getElementsByClassName('body-container')[0];
-    SimpleDataGrapherObject = new SimpleDataGrapher('first');
-    var value = '<%= current_user %>';
-    <% if current_user %>
-        SimpleDataGrapherObject.view.createButtons('yes');
-        var saveButton = SimpleDataGrapherObject.view.elementId + '_save_CSV';
-        var prevFile = SimpleDataGrapherObject.view.elementId + '_prev_file';
-        var saveFlag = false;
-        $('#' + saveButton).click(function(){
-            saveFlag = true;
-        });
+  var headerContainer = document.getElementsByClassName('body-container')[0];
+  SimpleDataGrapherObject = new SimpleDataGrapher('first');
+  var value = '<%= current_user %>';
+  <% if current_user %>
+    SimpleDataGrapherObject.view.createButtons('yes');
+    var saveButton = SimpleDataGrapherObject.view.elementId + '_save_CSV';
+    var prevFile = SimpleDataGrapherObject.view.elementId + '_prev_file';
+    var saveFlag = false;
+    $('#' + saveButton).click(function(){
+      saveFlag = true;
+    });
   $('#' + SimpleDataGrapherObject.view.plotGraphId).click(function(){
     if (saveFlag){
-      var arr = {};
-      arr['completeCsvMatrix'] = SimpleDataGrapherObject.view.csvParser.completeCsvMatrix;
-      arr['csvHeaders'] = SimpleDataGrapherObject.view.csvParser.csvHeaders;
-      arr['csvSampleData'] = SimpleDataGrapherObject.view.csvParser.csvSampleData;
-      arr['csvValidForYAxis'] = SimpleDataGrapherObject.view.csvParser.csvValidForYAxis;
-      arr['completeCsvMatrixTranspose'] = SimpleDataGrapherObject.view.csvParser.completeCsvMatrixTranspose;
-      let csvStringMatrix = SimpleDataGrapherObject.view.csvParser.completeCsvMatrixTranspose;
-      let csvStringForDownload = encodeURI('data:text/csv;charset=utf-8,' + csvStringMatrix.map(e => e.join(',')).join('\n'));
```

Comment:
Hi @lanefoxwood, please revert the changes on lines 26-33. Thanks!

Reply:
@TildaDares thanks, I will go ahead and revert those changes

Reply:
@TildaDares if you're busy and don't have time to help I understand, but I wanted to ask:
for reverting the changes I made to lines 26-33, what is the easiest way? I tried to edit the file in the pull request directly, but I don't think that's right. Would I have to put all of these commits into a new branch and make a new pull request or something? I'll keeping reading and trying to figure it out. Thanks for reading!

Reply:
Hi @lanefoxwood, I'm sorry if I wasn't clear enough but the original indentation on the file has the correct number of spaces, so we need to revert to lines 26-33 to how it was before. Lines 26-33 should be aligned with line 34. Thank you!!

Great job on figuring out how to add commits to an existing PR!

Reply:
Hi @lanefoxwood, do you need any help with this? Please let me know!